portal/src/locale-data/en.json

```diff
@@ -122,6 +122,7 @@
   "errors.bad-nft-collection": "This NFT collection is either not supported or is from another network.",
   "errors.alchemy-protocol": "We have some problem with this NFT collection. Please contact us at {ExternalLink, react, children{hello@authgear.com} href{mailto:hello@authgear.com}}.",
   "errors.deno-hook.typecheck": "{pre, react, children{{message}}}",
+  "errors.roles.key.validation.format": "Invalid role key. Only english alphbets, digits, colons (:) and underscores (_) are accepted.",
```

Comment:
The error message seems not correct when I enter key `123`

Reply:
@Ngkaokis This is actually because the regex in server side doesn't allow digits as the initial character. 
It seems differ from [the spec](https://github.com/authgear/authgear-server/pull/3556/files#diff-dff982576a46d7877aac0dfe34a729c8bee1d062c2e992a59093600d7502e922R22).
@louischan-oursky Should we allow digit as the first character?

Reply:
Use the implementation as the reference. Please update the spec and the error message to match the implementation.